provision/testdata/flavor_calico-3.23.2_tar/custom_resources_aci_calico.yaml

```diff
@@ -196,7 +196,7 @@ data:
                 "enable_updates": false
             },
             "aci_config": {
-                "system_id": "calico-l3out-fsvi-vlan-13",
+                "system_id": "calico-cni",
```

Comment:
why is this changing? are we sure the default type is getting set correctly to floating?

Reply:
yes, I saw this before committing. The change I made was for only type: floating and then I ran the fix-testdata.sh but that I believe changed the default. I will try again to see why this change is coming which is unrelated